ister_gui.py

```diff
@@ -2129,6 +2124,32 @@ def handler(self, config):
                 bundle for bundle in self.required_bundles
                 if not bundle.get('name') == 'telemetrics']
 
+        sysadmin_reqd = any(bundle['name'] == 'sysadmin-basic'
+                            for bundle in self.required_bundles)
+        sysadmin_basic = {'name': 'sysadmin-basic',
+                          'desc': 'Run common utilities useful for managing a '
+                                  'system'}
+        if config.get('Users') and not sysadmin_reqd:
```

Comment:
The options are `Create an administrative user` or `No user creation (login as root)`